sympy/functions/special/delta_functions.py

```diff
@@ -31,6 +31,20 @@ class DiracDelta(Function):
 
     5) ``DiracDelta(x,k) = 0``, for all ``x != 0``
 
+    Examples
```

Comment:
Create an explanation of what dirac delta function is and how it is different than normal functions in SymPy.